models/config.go

```diff
@@ -79,7 +80,7 @@ type ConfigChange struct {
 	Severity         string     `gorm:"column:severity" json:"severity"  faker:"oneof: critical, high, medium, low, info"`
 	Source           string     `gorm:"column:source" json:"source"`
 	Summary          string     `gorm:"column:summary;default:null" json:"summary,omitempty"`
-	Patches          string     `gorm:"column:patches;default:null" json:"patches,omitempty"`
```

Comment:
I don't think we want to remove the patch - This would still be useful for querying specific changes - the diff is in addition